common/network-common/src/main/java/org/apache/spark/network/protocol/Message.java

```diff
@@ -37,7 +37,8 @@
     ChunkFetchRequest(0), ChunkFetchSuccess(1), ChunkFetchFailure(2),
     RpcRequest(3), RpcResponse(4), RpcFailure(5),
     StreamRequest(6), StreamResponse(7), StreamFailure(8),
-    OneWayMessage(9), UploadStream(10), User(-1);
+    OneWayMessage(9), UploadStream(10), User(-1),
+    MergedBlockMetaRequest(11), MergedBlockMetaSuccess(12);
```

Comment:
nit. If you don't mind, shall we keep the order by adding this before `User(-1)`?

Reply:
Done so resolving